### DISCUSSION TITLE
Model download will not work ... just queued.

### DISCUSSION THREAD

**Discussion by schoenid** [BlafKing/sd-civitai-browser-plus#243] [Q&A]
Just as a remark for other people with this problem:

If the download will not work anymore (console reports: 'connection refused' as part of some messages):
Try to create a new` Personal CivitAI API key ` and enter it in the settings.

The last key I've created was no more working after 3 months of use.
To create a new one solved this problem.

**Comment by BlafKing** [comment]
Thanks for the heads up!
I'll see if I the API returns some information about an API key being invalid so that I can report it with a message to the end user! :)
